File: api/DTOs/Admin/Orders/OrderItemIdAndHRExecEmpNoDto.cs
namespace api.DTOs.Admin.Orders
{
    public class OrderItemIdAndHRExecEmpNoDto
    {
        public int OrderItemId { get; set; }
        public int HrExecEmpId { get; set; }
        public string HrExecUsername { get; set; }
        public string HrExecAppUserId { get; set; }
    }
}

File: client/src/app/_dtos/admin/orderItemIdAndHRExecEmpNoDto.ts
export interface IOrderItemIdAndHRExecEmpNoDto
{
    orderItemId: number;
    hrExecEmpId: number;
    hrExecUsername: string;
    hrExecAppUserId: number;
}

export class OrderitemidAndHRExecDetails implements IOrderItemIdAndHRExecEmpNoDto
{
    orderItemId=0;
    hrExecEmpId=0;
    hrExecUsername='';
    hrExecAppUserId=0;
}